planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -86,39 +97,78 @@
     public static final String SEACLOUDS_APPLICATION_POLICY_NAME = "seaclouds.app.information";
     public static final String SEACLOUDS_NODE_PREFIX = "seaclouds.nodes";
 
+    public static final String SEACLOUDS_APPLICATION_CONFIGURATION =
+            "seaclouds_configuration_policy";
+    public static final String SEACLOUDS_APPLICATION_CONFIGURATION_POLICY =
+            "configuration";
+    public static final String SEACLOUDS_MANAGEMENT_POLICY =
+            "eu.seaclouds.policy.SeaCloudsManagementPolicy";
+
+    public static final String INFLUXDB_DATABASE = "tower4clouds";
+    public static final String INFLUXDB_USERNAME = "root";
+    public static final String INFLUXDB_PASSWORD = "root";
+    public static final String GRAFANA_USERNAME = "admin";
+    public static final String GRAFANA_PASSWORD = "admin";
+
```

Comment:
@perezp 
It could be gotten from a config file but it is a first approach. We will try to improve in the next steps